src/main/main.c

```diff
@@ -581,7 +581,7 @@ int fd_listen(struct re_fhs **fhsp, re_sock_t fd, int flags, fd_h fh,
 		return EINVAL;
 	}
 
-	if (!fhsp || !flags || !fh)
+	if (!fhsp || !flags || fh == NULL)
```

Comment:
a explicit `NULL` check should not be needed

Reply:
Warning C4550: Expression evaluated as a function with missing parameter list by:vs2017. fh is a function pointer and  using !fh, which cause the compiler to warn. 

The above modifications have eliminated the warning